SPECS-EXTENDED/python-whoosh/python-whoosh.spec

```
@@ -62,6 +62,7 @@ rm -rf docs/html/.doctrees
 %py3_install
 
 %check
+%{__python3} -m pip install wheel
```

Wouldn't this be enough?

```suggestion
pip3 install wheel
```

---

Yes, I actually read a blog post: [Why should one use python -m pip](https://snarky.ca/why-you-should-use-python-m-pip/), which convinced me to use `python -m pip install ....`.